CalculatorTest/SeleniumTest/EpamRandomTests.cs

```diff
@@ -0,0 +1,56 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+using OpenQA.Selenium.Support.UI;
+
+namespace SeleniumTest;
+
+public class EpamRandomTests
+{
+    IWebDriver Driver;
+    private WebDriverWait Wait;
+    private string _epamUrl = "https://www.epam.com/";
+
+    [SetUp]
+    public void SetupBrowser()
+    {
+        Driver = new ChromeDriver();
+        var action = new Actions(Driver);
+        //Wait = new WebDriverWait(Driver, TimeSpan.FromSeconds(10));
+        Driver.Manage().Window.Maximize();
+        Driver.Navigate().GoToUrl(_epamUrl);
+        //Driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(10);
+        Driver.FindElement(By.Id("onetrust-accept-btn-handler")).Click();
```

Comment:
Before clicking to the AcceptAll button, you should wait until cookie banner is displayed.